opportunity_job_app/src/components/JobOfferDetails/JobOfferDetails.tsx

```diff
@@ -54,8 +65,26 @@ export default function JobOffersDetails({ jobOfferID }: JobOfferDetailsProps) {
                 const { account_id } = user;
                 await JobEnrollmentService.addJobEnrollment(jobOfferID, account_id);
                 fetchJobOffers();
+                setPopupDetails({
+                    popupVisibility: true,
+                    paragraphFirstText: 'Vasa prijava je uspeno prosledjena!',
+                    paragraphSecondText: 'Uskoro ce Vam se javiti neko iz organizacije ATINA',
+                    paragraphSecondVisibility: true,
+                    linkVisibility: true,
+                    linkText: 'Nazad na poslove',
+                    linkUrl: '/job-offers',
+                  });
             } catch (error) {
                 console.log("Enrollment error:", error);
+                    setPopupDetails({
+                    popupVisibility: true,
+                    paragraphFirstText: 'Error: Your enrollment could not be processed',
+                    paragraphSecondText: 'Please try again later',
```

Comment:
Ovaj tekst bih napisao na srpskom kako bi bili konzistenti sa ostatkom teksta u kodu (koji je sav na srpskom), bez obzira sto ce se sve to premestati u JSON fajlove za prevode.